src/drt/src/io/io_parser_helper.cpp

```diff
@@ -44,6 +44,88 @@ using namespace boost::polygon::operators;
 
 using Rectangle = boost::polygon::rectangle_data<int>;
 
+bool io::Parser::validWidth(odb::dbTechLayer* layer,
+                            int width,
```

Comment:
const method and width is a const argument

isValidWidth makes more sense for a predicate